### DISCUSSION TITLE
Will Infisical support LDAP in the future?

### DISCUSSION THREAD

**Discussion by GiamBoscaro** [Infisical/infisical#1188] [Feature Request]
Are you going to support LDAP auth in the future or is it in development maybe?

**Comment by maidul98** [comment]
LDAP support will be added after the release of new machine identity. Keep an eye out on our road map!

**Reply by gaby** [reply]
@maidul98 I noticed `LDAP` support was added recently but it's under the highest pricing tier, even for self-hosted. In contrast Hashi Vault only has `SAML` under their `enterprise` tier, everything else auth related is free.

https://developer.hashicorp.com/vault/docs/auth/ldap

**Reply by GiamBoscaro** [reply]
Well we already went with Vault months ago as LDAP was an hard requirement. Glad to have made the right choice.

**Reply by gaby** [reply]
I'm still deciding which one to use, and having at least LDAP for self-hosted would be useful.

SAML/SSO/PKI and those mega enterprise auth types make sense to be paid features.

**Reply by dangtony98** [reply]
Hi @gaby!

We did indeed decide to add `LDAP` to the enterprise tier (this may change in the future but for the time being that is the case). 

That said, you can definitely reach out to us or send in an email to obtain an enterprise license to unlock that feature.

**Reply by gaby** [reply]
@dangtony98 Was looking into using `Infisical` again and noticed the LDAP still shows as an enterprise feature. Has there been any plans to change that, or will stay like for the near future?
